.eslintrc.cjs

```diff
@@ -1,16 +1,5 @@
+require("@rushstack/eslint-patch/modern-module-resolution");
+
```

Comment:
Think this package `@rushstack/eslint-patch/modern-module-resolution` is unnecessary and can be removed

Reply:
done